.github/prompts/generate-custom-instructions-from-codebase.prompt.md

```
@@ -0,0 +1,240 @@
+---
+description: 'Migration and code evolution instructions generator for GitHub Copilot. Analyzes differences between two project versions (branches, commits, or releases) to create precise instructions allowing Copilot to maintain consistency during technology migrations, major refactoring, or framework version upgrades.'
+mode: 'agent'
+---
+
+# Migration and Code Evolution Instructions Generator
+
+## Configuration Variables
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)